### PR TITLE
Improve orb parameters

### LINKED ISSUE #73: Add a parameter for docker tar path
Making this customizable would be good:

https://github.com/codacy/codacy-plugins-test/blob/cb544a75ec81f99f9017fec19e10cc84978e1515/orbs/jobs/run_codacy_plugins_test.yml#L36

### PR DESCRIPTION
**Pull request by ljmf00**
Fixes #73 .

This PR also ignores other tests with `unless` keyword when running metrics tests.

### CODE REVIEW

File: orbs/jobs/run.yml [hunk 1]
```diff
@@ -31,6 +31,10 @@ parameters:
     type: string
     default: latest
     description: "Tool docker image tag"
+  docker_path:
```
Comment (lolgab) [inline orbs/jobs/run.yml]: What do you think on a more descriptive name? Like `docker_archive_path` or `docker_tar_path` or `docker_tar_archive_path`
it seems to be the path of the docker binary

File: orbs/jobs/run.yml [hunk 2]
```diff
@@ -44,42 +48,45 @@ steps:
...
+                  command: |
+                    codacy-plugins-test/$FILENAME pattern << parameters.docker_image_name >>:<< parameters.docker_image_tag >>
+        - when:
+            condition: << parameters.run_plugin_tests >>
+            steps:
+              - run:
+                  name: RunPluginTests
+                  command: |
+                    codacy-plugins-test/$FILENAME plugin << parameters.docker_image_name >>:<< parameters.docker_image_tag >>
+        - when:
```
Comment (lolgab) [inline orbs/jobs/run.yml]: This is non-default too, Should it be using `unless` too ?

Comment (ljmf00) [reply orbs/jobs/run.yml]: everything is inside `unless` whill be ignored. Are we using `multiple` test on `metrics`? If not, I guess we should ignore that too when using `metrics` test .

File: orbs/jobs/run.yml (hunk 1 above)
Comment (ljmf00) [reply orbs/jobs/run.yml]: I would say `docker_tar_path` what do you think?

**Comment by ljmf00** [comment]
> Use codacy convention for commit message and use `feature:` so it will bump minor version.

I'll use `feature` when I make the changes. Thanks :+1:

File: orbs/jobs/run.yml (hunk 2 above)
Comment (lolgab) [reply orbs/jobs/run.yml]: I did not get the indentation. So everything is inside the `run_metrics_tests`'s `unless`. Makes sense. 👍

File: orbs/jobs/run.yml (hunk 1 above)
Comment (lolgab) [reply orbs/jobs/run.yml]: `docker_tar_path` is fine.

**Comment by lolgab** [comment]
The problem with this PR is that at some point we will need to add `duplication` test. And it will break the semantic here

**Comment by lolgab** [comment]
I'm closing it since it will break when we'll add `duplication` tests to codacy-plugins-test

**Pull request closed without merging**